runtime/litmus/src/tests/mod.rs

```diff
@@ -24,19 +24,19 @@ pub mod setup {
 }
 
 mod transaction_payment {
-	use crate::{Call, Origin, Runtime, TransactionByteFee};
+	use crate::{Runtime, RuntimeCall, RuntimeOrigin, TransactionByteFee};
 
 	runtime_common::run_transaction_payment_tests!();
 }
 
 mod xcm_parachain {
 	use crate::{
 		tests::setup::{
-			Call as RelayCall, Origin as RelayOrigin, ParaA, ParaB, Relay, RelayChainRuntime,
-			TestNet,
+			ParaA, ParaB, Relay, RelayChainRuntime, RuntimeCall as RelayCall,
+			RuntimeOrigin as RelayOrigin, TestNet,
```

Comment:
So these name changes should also apply to common/src/tests/setup/relays.rs  ,   right? But no change apply to these files yet. Need double check.

Reply:
I compared the moonbeam's code, which is how they implemented it in version 0.9.30
<img width="586" alt="image" src="https://user-images.githubusercontent.com/87012234/201025566-c7dc0e11-602e-4673-a057-68c6f6622f32.png">

Reply:
check. Looks okay.